identity-aware-proxy/IAPClient/runTest.ps1

```diff
@@ -0,0 +1,16 @@
+# Copyright(c) 2018 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+
+dotnet restore
+dotnet run -- -c $env:TEST_IAP_CLIENT_ID -u $env:TEST_IAP_URI
```

Comment:
Am I missing something? Where is the test?

Reply:
If the run fails, it throws an exception, $LASTEXITCODE is non-zero, and our test script reports the test failed.

Reply:
Can we add a test that will call the original API that is behind a proxy using the sample code? The test will succeed if API returns http 200.

Reply:
I don't understand how the current test code differs from what you're requesting.

Reply:
The difference is that if you write it as a test, the output will be in test case format rather than console app format. Usually it's more convenient for presentation in build pipelines reports. Also for future, if you will make a change to the sample (method signature, etc), you will be forced to change the test too.